src/new.md

```diff
@@ -3,6 +3,29 @@ layout: page
 permalink: /resources/new/
 title: What's New
 ---
+## 5.7.1904
+
+- AddInData (Feature Preview) - Groups are now optional for AddInData objects, currently in Feature preview. Previously, groups were a required property for the AddInData object. This limited the potential usage of AddInData as there are situations where data should be available to all users regardless of scope, and some users were not able to access data when they belonged to groups outside the data’s scope. Removing this restriction means any user is now allowed to get an AddInData object if no group is specified for the object. Removing this restriction means any user is now allowed to get an AddInData object.
```

Comment:
Repeated comment at the end. remove last "Removing this restriction means any user is now allowed to get an AddInData object."

Reply:
Fixed